subgraphs/exchange/exchange.graphql

```diff
@@ -93,6 +93,106 @@ type SubsidizedSwapped @entity {
   path: [Bytes!]
 }
 
+# For LimitOrder contract
+type LimitOrderFilledByProtocol @entity(immutable: true) {
+  id: ID!
+  orderHash: Bytes! # bytes32
+  maker: Bytes! # address
+  taker: Bytes! # address
+  allowFillHash: Bytes! # bytes32
+  relayer: Bytes! # address
+  profitRecipient: Bytes! # address
+  fillReceipt_makerToken: Bytes! # address
+  fillReceipt_takerToken: Bytes! # address
+  fillReceipt_makerTokenFilledAmount: BigInt! # uint256
+  fillReceipt_takerTokenFilledAmount: BigInt! # uint256
+  fillReceipt_remainingAmount: BigInt! # uint256
+  fillReceipt_makerTokenFee: BigInt! # uint256
+  fillReceipt_takerTokenFee: BigInt! # uint256
```

Comment:
Is it possible to have a nested object? If not, could the naming be consistent to camelcase like `fillReceiptMakerToken`?

Reply:
No problem! The Graph official website does not clearly say that Entity can use nested structures.